client/src/components/Tooltips.tsx

```diff
@@ -46,9 +46,9 @@ export const AdviceTooltipIcon: React.FC<AdviceIconProps> = ({
   icon,
 }) => {
   const toolTip = [
-    <HStack px={2}>
+    <HStack key={text} px={2}>
```

Comment:
there isn't a size limit on how large the key can be, right? like, we aren't setting ourselves up for later failure of the code when someone puts the full text of war and peace by leo tolstoy into a tooltip.

Reply:
Good shout, I'll chop it down to like the first 12 characters